lib/manager.js

```diff
@@ -250,8 +250,8 @@ internals.Manager.prototype._loadHelpers = function (engine) {
                 try {
                     var helper = require(file);
                     if (typeof helper === 'function') {
-                        var offset = path.slice(-1) === Path.sep ? 0 : 1;
-                        var name = file.slice(path.length + offset, -3);
+                        var name = Path.basename(file);
+                        name = name.slice(0, -path.extname(name).length);
```

Comment:
I don't think that this will work correctly for helpers that are in subdirectories. Any reason to not use the original approach but just replace `-3` with `-Path.extname(file)`?